src/main/java/kr/codesquad/library/controller/BookController.java

```diff
@@ -56,22 +56,22 @@
     @GetMapping("/search")
     public ApiResult<BookSearchResponse> searchBooks(
             @RequestParam(value = "q") String title,
-            @RequestParam(value = "page", defaultValue = "1") @Min(PAGE_MINIMUM) int page) {
+            @RequestParam(value = "page", defaultValue = "1") @Min(PAGE_MINIMUM) Integer page) {
 
         return OK(bookService.searchBooks(title, page));
```

Comment:
```java
getCategoryBooks() -> bookService.findByCategoryId()
searchBooks() -> bookService.searchBooks()
```

위의 메소드에서는 bookService 이기에 가져오는 것이 Book 이라고 간주하여 암묵적으로 생략한 것 같고, 아래에서는 Book 을 표기했네요.
일관적이었으면 좋겠습니다.

```suggestion
        return OK(bookService.search(title, page));
```

Reply:
그렇네요 좀 일관성이 떨어지네요 이건 고쳐야할듯 감사합니다!